Tools/WinMLRunner/src/BindingUtilities.cpp

```diff
@@ -0,0 +1,1004 @@
+#include "d3dx12.h"
+#include <random>
+#include <time.h>
+#include "Windows.AI.Machinelearning.Native.h"
```

Comment:
Is this header inbox only?

Reply:
Good point! I fixed it so that the nuget version of native header is used when nuget configuration build is specified.